simple pg configuration

Ktechen opened the pull request:
SQL Database is now postgre

Ktechen committed:
Added pq to gem file

notefox commented on Gemfile:
```diff
@@ -44,7 +44,7 @@ gem "rouge"
 gem 'rails', '~> 6.1.3', '>= 6.1.3.2'
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3', '~> 1.4'
+gem 'pq'
```
wolltest du pg eingeben?
dieses hier?
https://rubygems.org/gems/pg/versions/0.18.4

Ktechen replied:
Oh, stimmt alles klar

notefox reviewed (changes requested):
naja fast ^^
ich glaube du hast da nen "kleinen" schreibfehler gemacht

Ktechen committed:
add pg | (not fixed version)

notefox commented on Gemfile:
```diff
@@ -43,8 +43,8 @@ gem "rouge"
-# Use sqlite3 as the database for Active Record
-gem 'pq'
+# Use PostgreSQL  as the database for Active Record
+gem 'pg', '~> 0.18.4'
```
nimm mal die neuste version

gem 'pg', '~> 1.2', '>= 1.2.3'
https://rubygems.org/gems/pg/versions/1.2.3

notefox reviewed (changes requested):
siehe Gemfile comment

Commit:
pg version change to:
 - gem 'pg', '~> 1.2', '>= 1.2.3'
 - Gemfile, Gemfile.lock automatic update

database.yml change to postgresql usage

add scripts for postgres docker setup:
 - scripts/run_local_docker.sh
 - scripts/setup_docker.sh

Commit:
run_local_docker.sh adjustment for better UX

notefox reviewed (approved):
works on my machine

Commit:
add function to script :
 - stop_local_postgres_server_if_running

MathisHTW reviewed (approved):
<3

Commit:
setup_docker.sh
 - postgresql-full -> postgresql-all

Commit:
setup_docker.sh
 - postgresql-full -> postgresql-all
